wiring/src/spark_wiring_string.cpp

```diff
@@ -217,7 +217,8 @@ String & String::copy(const char *cstr, unsigned int length)
 		return *this;
 	}
 	len = length;
-	strcpy(buffer, cstr);
+	memcpy(buffer, cstr, length);
+	buffer[len] = 0;
```

Comment:
They are the same thing, but I agree most people see a null terminator and know what's going on without thinking about it.  Sometimes the way to go though is to match the style used in the existing file instead of applying your own style here and there.  In that light, most of the file uses `= 0;` and there are only two instances of `= '\0';`